### PR TITLE
app icon for facelandmark

### PR DESCRIPTION
**Pull request by sukanvisapearyoo**
icon added for both ios and android no changes made to design of the app.

### CODE REVIEW

**Review by cdiddy77 — CHANGES_REQUESTED** [review]
Please get rid of the changes to `Podfile.lock` and, in the `objectdetection.xcodeproj`, the `project.pbxproj` since nothing really changed in those files.

File: examples/facelandmarkdetection/ios/Podfile.lock
Comment (sukanvisapearyoo) [inline examples/facelandmarkdetection/ios/Podfile.lock:1]: @cdiddy77 youre talking about this file? you want me to change it back to what it was (red)?

File: examples/objectdetection/ios/Podfile.lock
Comment (cdiddy77) [inline examples/objectdetection/ios/Podfile.lock:1]: this should be reverted from the commit

File: examples/objectdetection/ios/objectdetection.xcodeproj/project.pbxproj
Comment (cdiddy77) [inline examples/objectdetection/ios/objectdetection.xcodeproj/project.pbxproj:1]: this should be reverted from the PR

**Review by cdiddy77 — APPROVED** [review]
just remove the changes to objectdetection files and you are good

File: examples/objectdetection/ios/Podfile.lock
Comment (sukanvisapearyoo) [reply examples/objectdetection/ios/Podfile.lock:1]: I believe i have reverted it

**Review by cdiddy77 — APPROVED** [review]

**Pull request merged**